Doc/index.ts

```diff
@@ -0,0 +1,30 @@
+import { AzureFunction, Context } from "@azure/functions"
+import Jsdoc from "swagger-jsdoc"
+
+const swaggerJsdoc: AzureFunction = async function (context: Context): Promise<void> {
+  const options = {
+    swaggerDefinition: {
+      openapi: "3.0.0",
+      info: {
+        title: "Data MarketPlace API",
+        version: "1.0.0",
+      },
+    },
+    apis: [
+      "./GetAssetTrigger/index.ts",
+      "./GetMaintainersTrigger/index.ts",
+      "./GetPopularAssetTrigger/index.ts",
+      "./GetTermsAndConditionsTrigger/index.ts",
+    ],
+  }
+
+  context.res = {
+    status: 200,
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: Jsdoc(options),
+  }
+}
```

Comment:
not sure whether i just don't understand swagger, but to me, it seems like this file should be imported as part of a script that, when called, generates static html files to be served by some endpoint

Reply:
I did not understand this comment completely.  It is how it generated on back end.